### ISSUE TITLE
Server crashes during world tick because of RainbowChicken

### ISSUE THREAD

**Issue by lunofe** [jayhillx/MysticsBiomes#23]
Hm, déjà vu? 😅

- Minecraft: 1.20.1
- Forge: 47.2.19
- MysticsBiomes: 1.20.1-3.1.6

Crashreport:
```
Description: Exception ticking world
java.lang.NoClassDefFoundError: org/apache/commons/compress/utils/Lists
	at com.mysticsbiomes.common.animal.RainbowChicken.<init>(RainbowChicken.java:50) ~[mysticsbiomes-1.20.1-3.1.6.jar%23292!/:3.1.6] {re:classloading}
	at net.minecraft.world.entity.EntityType.m_20615_(EntityType.java:552) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:mixin,xf:fml:forge:forge_method_redirector,re:classloading,xf:fml:forge:forge_method_redirector,pl:mixin:APP:architectury-common.mixins.json:inject.MixinEntityType,pl:mixin:A}
	at net.minecraft.world.level.NaturalSpawner.m_220450_(NaturalSpawner.java:380) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:classloading,xf:fml:forge:forge_method_redirector}
	at net.minecraft.world.level.levelgen.NoiseBasedChunkGenerator.m_6929_(NoiseBasedChunkGenerator.java:453) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.level.chunk.ChunkStatus.m_196757_(ChunkStatus.java:107) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.world.level.chunk.ChunkStatus$SimpleGenerationTask.m_214024_(ChunkStatus.java:281) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:classloading}
	at net.minecraft.world.level.chunk.ChunkStatus.m_280308_(ChunkStatus.java:217) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ChunkMap.lambda$scheduleChunkGeneration$27(ChunkMap.java:667) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:lithium.mixins.json:world.player_chunk_tick.ThreadedAnvilChunkStorageMixin,pl:mixin:A}
	at com.mojang.datafixers.util.Either$Left.map(Either.java:38) ~[datafixerupper-6.0.8.jar%23164!/:?] {}
	at net.minecraft.server.level.ChunkMap.lambda$scheduleChunkGeneration$29(ChunkMap.java:658) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:lithium.mixins.json:world.player_chunk_tick.ThreadedAnvilChunkStorageMixin,pl:mixin:A}
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?] {}
	at net.minecraft.server.level.ChunkTaskPriorityQueueSorter.m_143188_(ChunkTaskPriorityQueueSorter.java:62) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:classloading}
	at net.minecraft.util.thread.ProcessorMailbox.m_18759_(ProcessorMailbox.java:91) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:classloading}
	at net.minecraft.util.thread.ProcessorMailbox.m_18747_(ProcessorMailbox.java:146) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:classloading}
	at net.minecraft.util.thread.ProcessorMailbox.run(ProcessorMailbox.java:102) ~[server-1.20.1-20230612.114412-srg.jar%23332!/:?] {re:classloading}
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {re:mixin,re:mixin,re:mixin}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin,re:mixin}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {re:mixin,re:mixin,re:mixin}
Caused by: java.lang.ClassNotFoundException: org.apache.commons.compress.utils.Lists
	... 22 more
```

**Comment by niftybottle** [comment]
Also encountering this error

**Comment by jayhillx** [comment]
this is now fixed in version 3.2.
